blueprint/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Scripts to screenpop a URL when after inactivity when a conversation ends on Genesys Cloud
+author: zino.onokpise
+indextype: blueprint
+icon: blueprint
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint explains how to create an embeddable app in scripts to set up a notification channel and listen for events. In this blueprint, a subscription is made to a user's active conversations. Events from this topic are then used to track the state of the conversation and alerts to the user when a conversation has ended and the chat should be closed
```

Comment:
This Genesys Cloud Developer Blueprint explains how to create an embeddable app in scripts to set up a notification channel and listen for events. In this blueprint, a subscription is made to a user's active conversations. Events from this topic are then used to track the state of the conversation and alerts to the user when a conversation has ended and the chat should be closed. -> 

This Genesys Cloud developer blueprint explains how to create an embeddable app that runs within a Genesys Cloud agent script.  This embeddable app will set up a notification and listen for messaging conversation events from an agent's active messaging conversation.  These events will be used to track the state of a messaging conversation and when a conversation has been inactive the agent will be notified.  

*(JUST MY THOUGHTS :) )

Reply:
Sounds better